test/test_tree_3d_mhd.jl

```diff
@@ -291,6 +291,20 @@ end
         @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
     end
 end
+
+  @trixi_testset "elixir_mhd_diffusive_alfven_wave.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_diffusive_alfven_wave.jl"),
+      l2 = [3.470701487566952e-5, 0.00037320266655784104, 0.0001814791326953306, 0.0004187613798734842, 0.00011587163989250743, 0.00037032036399761947, 0.00018270779781144185, 0.00041229458967006154, 2.7753210639128732e-5],
+      linf = [0.0001348833811495309, 0.000892691474205648, 0.00043846917904940587, 0.0010014206836504308, 0.000443288547495424, 0.0008149498186467707, 0.00042111533550182045, 0.000918156282098577, 0.00010109055017304109])
+  end
```

Comment:
Here we definitely should check for allocations as for the other tests.

Reply:
The memory issue has been fixed.